app/controllers/catalog_controller.rb

```diff
@@ -208,76 +229,40 @@ class CatalogController < ApplicationController
     # mean") suggestion is offered.
     config.spell_max = 5
 
+    # Nav actions from Blacklight
+    config.add_nav_action(:bookmark, partial: 'blacklight/nav/bookmark', if: :render_bookmarks_control?)
+    config.add_nav_action(:search_history, partial: 'blacklight/nav/search_history')
+    config.add_nav_action(:help, partial: 'shared/help')
+
+    # Tools from Blacklight
+    config.add_results_collection_tool(:sort_widget)
+    config.add_results_collection_tool(:per_page_widget)
+    config.add_show_tools_partial(:bookmark, partial: 'bookmark_control', if: :render_bookmarks_control?)
+    config.add_show_tools_partial(:email, callback: :email_action, validator: :validate_email_params)
+
     # Custom tools for GeoBlacklight
-    #config.add_show_tools_partial :web_services, if: proc { |_context, _config, options| options[:document] && (Settings.WEBSERVICES_SHOWN & options[:document].references.refs.map(&:type).map(&:to_s)).any? }
-    config.add_show_tools_partial :help, partial: 'help'
     config.add_show_tools_partial :metadata, if: proc { |_context, _config, options| options[:document] && (Settings.METADATA_SHOWN & options[:document].references.refs.map(&:type).map(&:to_s)).any? }
-    config.add_show_tools_partial :exports, partial: 'exports', if: proc { |_context, _config, options| options[:document] }
-    config.add_show_tools_partial :data_dictionary, partial: 'data_dictionary', if: proc { |_context, _config, options| options[:document] }
-    config.add_show_tools_partial :downloads, partial: 'downloads', if: proc { |_context, _config, options| options[:document] }
+    config.add_show_tools_partial :carto, partial: 'carto', if: proc { |_context, _config, options| options[:document] && options[:document].carto_reference.present? }
+    config.add_show_tools_partial :arcgis, partial: 'arcgis', if: proc { |_context, _config, options| options[:document] && options[:document].arcgis_urls.present? }
+    config.add_show_tools_partial :data_dictionary, partial: 'data_dictionary', if: proc { |_context, _config, options| options[:document] && options[:document].data_dictionary_download.present? }
 
     # Configure basemap provider for GeoBlacklight maps (uses https only basemap
     # providers with open licenses)
     # Valid basemaps include:
-    # 'positron' http://cartodb.com/basemaps/
-    # 'darkMatter' http://cartodb.com/basemaps/
+    # 'positron'
+    # 'darkMatter'
+    # 'positronLite'
+    # 'worldAntique'
+    # 'worldEco'
+    # 'flatBlue'
+    # 'midnightCommander'
+    # 'openstreetmapHot'
+    # 'openstreetmapStandard'
+
     config.basemap_provider = 'osmhot'
```

Comment:
Can you confirm this should use this value and not `openstreetmapHot` ? Using a value not in the valid list makes me nervous :)

Reply:
Yeah, the OSM basemaps got added to GBL recently. You can also delete https://github.com/MITLibraries/geoweb/blob/gbl-upgrade-oops/app/assets/javascripts/geoweb.js#L6-L11 now.

Reply:
Updated. Thanks for catching this Jeremy, and thanks Mike for clarifying!